content/blog/2022/cq-syntax.md

```diff
@@ -38,6 +39,18 @@ it is very unlikely that there will be
 any breaking changes.
 {% endcallout %}
 
+{% set update = ['Update', utility.datetime('2022-09-01')] | join(' ') %}
+{% callout 'note', update %}
+- Chrome 105 shipped on August 30,
+  with support for Container Queries and units.
+- The Safari bug around negated queries
+  has been fixed in
+  [Safari Technology Preview 152](https://webkit.org/blog/13137/release-notes-for-safari-technology-preview-152/).
+- I expect Safari 16 to release
+  [during the Apple event](https://www.apple.com/apple-events/)
+  on September 7.
```

Comment:
@mirisuzanne I'm not sure this happened?

Reply:
It didn't. Oops. Guess I should update.